app/domain/authentication/authn_jwt/consts.rb

```diff
@@ -6,7 +6,7 @@ module AuthnJwt
     PROVIDER_URI_RESOURCE_NAME = "provider-uri"
     JWKS_URI_RESOURCE_NAME = "jwks-uri"
     ISSUER_RESOURCE_NAME = "issuer"
-
+    AUTHN_JWT_RESOURCE_PREFIX = "cucumber:variable:conjur/authn-jwt"
```

Comment:
`authn-jwt` should be fetched from authenticator_name param because maybe in the future this logic will serve other authenticators

Reply:
Removed const and will build this id path in the regex params